### PR TITLE
[DOC] Add Pyroscope to list of products

### PR DESCRIPTION
**Pull request by knylander-grafana**
**What is this feature?**

Adds a brief description of Pyroscope to the summary of telemetry signals in the fundamentals/index doc. This update is in a shared file. 

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [X] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/contribute/release-notes/#how-to-determine-if-content-belongs-in-whats-new), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/contribute/release-notes/) doc.

### CODE REVIEW

File: docs/sources/shared/basics/what-is-grafana.md [hunk 1]
```diff
@@ -12,16 +12,20 @@ Grafana open source is open source visualization and analytics software. It allo
 
 ### Grafana Loki
 
-Grafana Loki is an open source, set of components that can be composed into a fully featured logging stack. For more information, refer to [Loki documentation](/docs/loki/latest/).
+Grafana Loki is an open source, set of components that can be composed into a fully featured logging stack. For more information, refer to [Loki documentation](https://grafana.com/docs/loki/<LOKI_VERSION>/).
```
Comment (chri2547) [inline docs/sources/shared/basics/what-is-grafana.md:15]:
```suggestion
Grafana Loki is an open-source set of components that can be composed into a fully featured logging stack. For more information, refer to [Loki documentation](https://grafana.com/docs/loki/<LOKI_VERSION>/).
```

File: docs/sources/shared/basics/what-is-grafana.md [hunk 2]
```diff
@@ -12,16 +12,20 @@ Grafana open source is open source visualization and analytics software. It allo
...
+Grafana Tempo is an open source, easy-to-use and high-volume distributed tracing backend. For more information, refer to [Tempo documentation](https://grafana.com/docs/tempo/<TEMPO_VERSION>/).
 
 ### Grafana Mimir
 
-Grafana Mimir is an open source software project that provides a scalable long-term storage for Prometheus. For more information about Grafana Mimir, refer to [Grafana Mimir documentation](/docs/mimir/latest/).
+Grafana Mimir is an open source software project that provides a scalable long-term storage for Prometheus. For more information about Grafana Mimir, refer to [Grafana Mimir documentation](https://grafana.com/docs/mimir/<MIMIR_VERSION>/).
+
+### Grafana Pyroscope
+
+Grafana Pyroscope is an open source software project for aggregating continuous profiling data. Continuous profiling is an observability signal that helps you understand your workload’s resources usage down. For more information, refer to [Grafana Pyroscope documentation](https://grafana.com/docs/pyroscope/<PYROSCOPE_VERSION>/).
```
Comment (bryanhuhta) [inline docs/sources/shared/basics/what-is-grafana.md:27]:
```suggestion
Grafana Pyroscope is an open source software project for aggregating continuous profiling data. Continuous profiling is an observability signal that helps you understand your workload’s resources usage. For more information, refer to [Grafana Pyroscope documentation](https://grafana.com/docs/pyroscope/<PYROSCOPE_VERSION>/).
```

I assume "down" shouldn't belong?

**Review by bryanhuhta — APPROVED** [review]
The Pyroscope portion LGTM :shipit:

**Review by marcsanmi — APPROVED** [review]

File: docs/sources/shared/basics/what-is-grafana.md (hunk 1 above)
Comment (knylander-grafana) [reply docs/sources/shared/basics/what-is-grafana.md:15]: Thanks, Chris!

File: docs/sources/shared/basics/what-is-grafana.md (hunk 2 above)
Comment (knylander-grafana) [reply docs/sources/shared/basics/what-is-grafana.md:27]: Yes. Good catch!

**Review by chri2547 — APPROVED** [review]

**Pull request merged**